core/location/gendb/loader.go

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2017 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package gendb
+
+import (
+	"compress/gzip"
+	"encoding/base64"
+	"errors"
+	"io"
+	"io/ioutil"
+	"strings"
+)
+
+// EncodedDataLoader returns emmbeded database as byte array
+func EncodedDataLoader(data string, originalSize int, compressed bool) (decompressed []byte, err error) {
+	var reader io.Reader
```

Comment:
This can be just `reader := base64.NewDecoder(base64.RawStdEncoding, strings.NewReader(data))`

Reply:
It cannot. Because of the compressed flag, reader can be further decorated with gzip.Reader

Reply:
Sorry. Got your point. It can be just reader := ...
Thanks.